components/builder-web/app/banner/banner.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="banner-component" *ngIf="hidden">
+<div class="banner-component" [class.hidden]="!hidden">
```

Comment:
I thought it was more efficient to use ngIf here since it actually removes this node from the DOM.

Reply:
This is purely for aesthetics, to animate the banner rather dismissal rather than just removing it from the DOM. But you're right, it still stays. Just something I figured I might try to see what folks thought.

Reply:
Here's what it looks like: 

![](https://d3vv6lp55qjaqc.cloudfront.net/items/0W0M0B1b15083O0g3X0a/Screen%20Recording%202018-01-05%20at%2010.21%20AM.gif?X-CloudApp-Visitor-Id=1466146&v=bb157f3c)

Reply:
💯  the small things make a big difference.